org-formation/075-security-hub/security-hub-suppress-lambda.yaml

```diff
@@ -0,0 +1,119 @@
+---
+Transform: AWS::Serverless-2016-10-31
+Resources:
+  AWSLambdaPowertoolsApplication:
+    Type: AWS::Serverless::Application
+    Properties:
+      Location:
+        ApplicationId: arn:aws:serverlessrepo:eu-west-1:057560766410:applications/aws-lambda-powertools-python-layer
+        SemanticVersion: 1.17.0
```

Comment:
Do we need to use this version? It looks like the current version is 2.4.0: https://awslabs.github.io/aws-lambda-powertools-python/latest/

Reply:
Updated to 2.5.0